ui/src/router/components/configuration/components/experiment_config_section/variables/VariablesConfigTable.js

```diff
@@ -7,7 +7,6 @@ import {
   EuiTitle,
 } from "@elastic/eui";
 import orderBy from "lodash/orderBy";
-import "./VariablesConfigTable.scss";
```

Comment:
The removal of the styles in this file, together with the setting of the column widths manually to have the same ratio via the `columnWidths` prop, realigns the table to fit with the other router configs table just above it.

Before:
![Screenshot 2024-07-15 at 17 46 18](https://github.com/user-attachments/assets/07ca68f9-a334-4021-91e5-fa50d247a938)

After:
![Screenshot 2024-07-15 at 18 11 29](https://github.com/user-attachments/assets/8bc3bf52-0650-420a-bb11-a512799a6c4d)